src/js/logic.js

```diff
@@ -0,0 +1,33 @@
+import { storyObj } from "./index";
```

Comment:
Everything comes typically together in index.js. As far as I know it is not the best approach to import something from index.js. If the gameLogic module needs the story object it should be injected into the gameLogic module in the index.js. I could try to adjust it, if you like. Or you try it. The steps should be something like this: 

1. Have a story object as parameter for gameLogic module
2. Don't invoke the gameLogic module directly in its own module
3. Import gameLogic into index.js
4. Create the story object in index.js
5. Invoke the gameLogic in index.js with the story object
6. Problem is probably that also import of gameLogic into chapterDOM doesn't work anymore: need to be done also in index.js with the instantiated

Reply:
I agree that importing something from index module is bad practice and we should avoid it. I rewritten code following your directions and... it works. I needed to add new parameter to chapterDOM which is callback function which is used in makeDroppable. So, when we create story and then structure for first chapter, we need to remember to pass displayChapterEnd as second argument (or other method in the future).

But most important thing that i realized thanks to your remarks is the way it works now - it just sucks. As i mentioned above, we need to create story object, then structure for first chapter, then starting page and at the end - render starting page. It isn't a big problem for just one story, but if we want add more stories in the future, we will be forced to write more unnecessary code to create mentioned elements for each story. This is my idea to solve this problem:
1. gameLogic will be invoked in index.js.
2. I will change this function to take one parameter - array of stories.
3. gameLogic will take care of generating first chapter page for every story and it will do it automatically. I'm not 100% sure if this function should also generate starting page.
4. gameLogic should also take care of rendering content of our game when specified requirements are met (just like goToNextChapter method).

That's my ideas to solve this problem for know. Let me know what do you think about this approach and of course your ideas to solve this problem @Tudwall  @emdevs

Reply:
> * gameLogic will be invoked in index.js.
> * I will change this function to take one parameter - array of stories.
> * gameLogic will take care of generating first chapter page for every story and it will do it automatically. I'm not 100% sure if this function should also generate starting page.
> * gameLogic should also take care of rendering content of our game when specified requirements are met (just like goToNextChapter method).

Sounds like a great structure to me. I would put it as an improvement as extra task and leave it for now - is probably not really much extra effort, if this is adjusted later, when we start having more stories. And now first put more focus on improving the flow and details of our first story.